test/workarea/paypal/ext/active_merchant/paypal_express_gateway_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+module ActiveMerchant
+  module Billing
+    class PaypalExpressGatewayTest < Workarea::TestCase
+      def test_add_button_source
+        xml = Builder::XmlMarkup.new
```

Comment:
Xml Builder is what's used in the gateway: https://github.com/activemerchant/active_merchant/blob/1648fed6eb5cee78878d08e7418a2a44475c044e/lib/active_merchant/billing/gateways/paypal/paypal_common_api.rb#L312